src/controller.coffee

```diff
@@ -2204,10 +2223,8 @@ Editor::showDropdown = ->
       div.innerHTML = el.display
       div.className = 'droplet-dropdown-item'
 
-      # Match fonts
-      div.style.fontFamily = @fontFamily
-      div.style.fontSize = @fontSize
       div.style.paddingLeft = helper.DROPDOWN_ARROW_WIDTH
+      div.style.paddingRight = DROPDOWN_SCROLLBAR_PADDING
```

Comment:
I could do that, but it would require letting the browser render first (to see what the actual height is -- we can't compute this, because embedders can put arbitrary html in the dropdown choices), which means users would see a flash.